### PR TITLE
Example for multiple dependencies added

### PR DESCRIPTION
**Pull request by hmol**

### CODE REVIEW

**Comment by gep13** [comment]
@hmol thank you for making this addition!  Are you in a position to sign our CLA?

**Comment by gep13** [comment]
@hmol normally the CLA Bot comes back and updates the PR if the CLA has indeed been signed.

@dnfgituser @dnfclas @jongalloway can you please confirm if the CLA has indeed been signed?

**Comment by gep13** [comment]
@dnfclas This seems to be the second time that this has happened.  Is there an issue that we need to be aware of?  Thanks!

**Comment by dnfgituser** [comment]
@gep13 There was an issue where we were not getting the CLAs after they were signed, which also prevented the labels from getting updated. This has been fixed and the process is working again. Sorry for the trouble!

**Comment by gep13** [comment]
@dnfgituser thanks for getting back to me.

File: input/docs/fundamentals/dependencies.md [hunk 1]
```diff
@@ -22,11 +22,35 @@ RunTarget("B");
...
+});
+
+Task("B")
+    .Does(() =>
+{
+});
+
+Task("C")
+    .IsDependentOn("A")
+    .IsDependentOn("A")
```
Comment (pitermarx) [inline input/docs/fundamentals/dependencies.md]: shouldn't this be "B"?

**Comment by gep13** [comment]
@hmol this was rebased and merged here: https://github.com/cake-build/website/commit/782ec17a40e6e920f92edf5842b77f5140074c8d

Thank you for your contribution!

**Pull request closed without merging**